src/components/core/button/toggle-button.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { useTranslation } from '@/lib/i18n';
+
+
+const ToggleButton = ({ isActive = false, onClick = () => {}, label = '', ariaLabel = '' }) => {
+  return (
+    <button
+      className={cn('py-2 px-4 rounded-md', isActive ? 'bg-cyan text-white' : 'bg-white text-cyan')}
+      onClick={onClick}
+      aria-label={ariaLabel}
+      aria-pressed={isActive}
+    >
+      {label}
+    </button>
+  );
+};
+
+ToggleButton.propTypes = {
+  isActive: PropTypes.bool,
+  onClick: PropTypes.func,
+  label: PropTypes.string,
+  ariaLabel: PropTypes.string,
+};
+
+const ToggleButtonGroup = ({ options = [], activeOption = '', onOptionChange = () => {}, labelPrefix = '' }) => {
+    const t = useTranslation('home');
+    return (
+    <>
+      {options.map((option) => (
+        <ToggleButton
+          key={option}
+          isActive={activeOption === option}
+          onClick={() => onOptionChange(option)}
+          label={t(`${labelPrefix}.${option}`)}
```

Comment:
So, you are combining i18n files, "label" and "labelPrefix," to get the label, correct? 

Since this is a core component that should be pure and composable, please avoid involving i18n here. You could define the option as `{ label: String, value: String | Number }`. The label should be obtained directly from the parent component.